mne/dipole.py

```diff
@@ -214,6 +227,70 @@ def __len__(self):
         return self.pos.shape[0]
 
 
+class DipoleFixed(object):
+    """Dipole class for fixed-position dipole fits
+
+    Parameters
+    ----------
+    fname : str
+        The file to load.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    See Also
+    --------
+    Dipole
+
+    Notes
+    -----
+    This class is for sequential dipole fits, where the position
+    changes as a function of time. For fixed dipole fits, where the
+    position is fixed as a function of time, use :class:`mne.DipoleFixed`.
+    """
+    @verbose
+    def __init__(self, fname, verbose=None):
+        _check_fname(fname, overwrite=True, must_exist=True)
+        logger.info('Reading %s ...' % fname)
+        self.info, self.nave, self._aspect_kind, self.first, self.last, \
+            self.comment, self.times, self.data = _read_evoked(fname)
+        self.kind = _aspect_rev.get(str(self._aspect_kind), 'Unknown')
+        self.verbose = verbose
+
+    @property
+    def ch_names(self):
+        return self.info['ch_names']
+
+    @verbose
+    def save(self, fname, verbose=None):
+        """Save dipole in a .fif file
+
+        Parameters
+        ----------
+        fname : str
+            The name of the .fif file.
```

Comment:
"It is recommended to end the file with -dip.fif to make it explicit that the file contains dipole information."